src/codegen/compile.ml

```diff
@@ -4395,6 +4452,23 @@ module MakeSerialization (Strm : Stream) = struct
       by the next GC.
   *)
 
+  let register_delayed_globals env =
+    (E.add_global32_delayed env "__typtbl" Immutable,
+     E.add_global32_delayed env "__typtbl_end" Immutable,
+     E.add_global32_delayed env "__typtbl_size" Immutable,
+     E.add_global32_delayed env "__typtbl_idltyps" Immutable)
```

Comment:
Hmm, it would be good to have a `Note` somewhere, referenced from here and other places, that explains the big picture: What is the meaning of these globals, how are they constructed, etc.

Reply:
Done

Reply:
Done